src/TypeScript.UnitTests/Rules/SampleRuleMetadata.json

```diff
@@ -0,0 +1,56 @@
u003e\\n\\u003cp\\u003e\\u003ca href\\u003d'https://www.npmjs.com/package/csurf'\\u003ecsurf\\u003c/a\\u003e module:\\u003c/p\\u003e\\n\\u003cpre\\u003e\\nconst cookieParser \\u003d require(\\u0027cookie-parser\\u0027);\\nconst csrf \\u003d require(\\u0027csurf\\u0027);\\nconst express \\u003d require(\\u0027express\\u0027);\\n\\nlet csrfProtection \\u003d csrf({ cookie: { secure: false }}); // Sensitive\\n\\u003c/pre\\u003e\\n\\u003ch2\\u003eCompliant Solution\\u003c/h2\\u003e\\n\\u003cp\\u003e\\u003ca href\\u003d'https://www.npmjs.com/package/cookie-session'\\u003ecookie-session\\u003c/a\\u003e module:\\u003c/p\\u003e\\n\\u003cpre\\u003e\\nlet session \\u003d cookieSession({\\n  secure: true,// Compliant\\n});  // Compliant\\n\\u003c/pre\\u003e\\n\\u003cp\\u003e\\u003ca href\\u003d'https://www.npmjs.com/package/express-session'\\u003eexpress-session\\u003c/a\\u003e module:\\u003c/p\\u003e\\n\\u003cpre\\u003e\\nconst express \\u003d require(\\u0027express\\u0027);\\nconst session \\u003d require(\\u0027express-session\\u0027);\\n\\nlet app \\u003d express();\\napp.use(session({\\n  cookie:\\n  {\\n    secure: true // Compliant\\n  }\\n}));\\n\\u003c/pre\\u003e\\n\\u003cp\\u003e\\u003ca href\\u003d'https://www.npmjs.com/package/cookies'\\u003ecookies\\u003c/a\\u003e module:\\u003c/p\\u003e\\n\\u003cpre\\u003e\\nlet cookies \\u003d new Cookies(req, res, { keys: keys });\\n\\ncookies.set(\\u0027LastVisit\\u0027, new Date().toISOString(), {\\n  secure: true // Compliant\\n}); // Compliant\\n\\u003c/pre\\u003e\\n\\u003cp\\u003e\\u003ca href\\u003d'https://www.npmjs.com/package/csurf'\\u003ecsurf\\u003c/a\\u003e module:\\u003c/p\\u003e\\n\\u003cpre\\u003e\\nconst cookieParser \\u003d require(\\u0027cookie-parser\\u0027);\\nconst csrf \\u003d require(\\u0027csurf\\u0027);\\nconst express \\u003d require(\\u0027express\\u0027);\\n\\nlet csrfProtection \\u003d csrf({ cookie: { secure: true }}); // Compliant\\n\\u003c/pre\\u003e\\n\\u003ch2\\u003eSee\\u003c/h2\\u003e\\n\\u003cul\\u003e\\n  \\u003cli\\u003e \\u003ca href\\u003d'https://www.owasp.org/index.php/Top_10-2017_A3-Sensitive_Data_Exposure'\\u003eOWASP Top 10 2017 Category A3\\u003c/a\\u003e - Sensitive Data Exposure\\n  \\u003c/li\\u003e\\n  \\u003cli\\u003e \\u003ca href\\u003d'http://cwe.mitre.org/data/definitions/311'\\u003eMITRE, CWE-311\\u003c/a\\u003e - Missing Encryption of Sensitive Data \\u003c/li\\u003e\\n  \\u003cli\\u003e \\u003ca href\\u003d'http://cwe.mitre.org/data/definitions/315'\\u003eMITRE, CWE-315\\u003c/a\\u003e - Cleartext Storage of Sensitive Information in a Cookie \\u003c/li\\u003e\\n  \\u003cli\\u003e \\u003ca href\\u003d'http://cwe.mitre.org/data/definitions/614'\\u003eMITRE, CWE-614\\u003c/a\\u003e - Sensitive Cookie in HTTPS Session Without \\u0027Secure\\u0027 Attribute \\u003c/li\\u003e\\n  \\u003cli\\u003e \\u003ca href\\u003d'https://www.sans.org/top25-software-errors/#cat3'\\u003eSANS Top 25\\u003c/a\\u003e - Porous Defenses \\u003c/li\\u003e\\n\\u003c/ul\\u003e",
```

Comment:
We don't currently need the HTML description so I haven't bothered loading and converting it.